deeplearning4j-scaleout/spark/dl4j-spark/src/main/java/org/deeplearning4j/spark/impl/graph/SparkComputationGraph.java

```diff
@@ -632,10 +696,43 @@ public Evaluation evaluate(JavaRDD<DataSet> data, List<String> labelsList, int e
      * @param <T>             Type of evaluation instance to return
      * @return                IEvaluation instance
      */
+    @SuppressWarnings("unchecked")
     public <T extends IEvaluation> T doEvaluation(JavaRDD<DataSet> data, T emptyEvaluation, int evalBatchSize) {
+        IEvaluation[] arr = new IEvaluation[]{emptyEvaluation};
+        return (T)doEvaluation(data, evalBatchSize, arr)[0];
+    }
+
+    /**
+     * Perform distributed evaluation on a <i>single output</i> ComputationGraph form DataSet objects using Spark.
+     * Can be used to perform multiple evaluations on this single output (for example, {@link Evaluation} and
```

Comment:
Link shows up correctly once rendered in javadoc (due to import of Evaluation class)